e2e/js_image/smoketest.yaml

```diff
@@ -0,0 +1,7 @@
+schemaVersion: 2.0.0
+
+commandTests:
+    - name: 'smoke'
+      command: 'bash'
+      args: ['/app/main.sh']
```

Comment:
kinda strange that the user API here is a bash script, when the app only had JS and no bash - another sign that the added launcher here should go away

Reply:
even without the launcher script, this would have been a bash script since js_binary creates one to do bunch of path logic and env stuff.